packages/nextjs/app/page.tsx

```diff
@@ -10,7 +10,15 @@ import { useAccount } from "wagmi";
 
 const Home: NextPage = () => {
   const { address: connectedAddress } = useAccount();
-
+  const [isCheckInActive, setCheckInActive] = useState(false);
+  const handleCheckIn = () => {
+    console.log("Moving on to the next step!");
+    // Display check-in form
```

Comment:
we can choose to display the form with a useState and manage visibility like that, it could be messy quick, and force us to have navigation inside the component
, or use a "next-link" (https://nextjs.org/docs/pages/api-reference/components/link and built in routing: https://nextjs.org/docs/app/building-your-application/routing) to manage visibility and navigation.

(alternative to reactRouter) I'll try to set up the navigation scoffolding so we have that in place already 👍

Reply:
OK, I will need to get smart on React routing, my plan is to read through the NextJS documentation soon.